can_tools/scrapers/official/DE/de_vaccine.py

```diff
@@ -141,7 +141,7 @@ def normalize(self, data: Dict[str, requests.models.Response]) -> pd.DataFrame:
                 timezone="US/Eastern",
             )
             .assign(
-                vintage=self._retrieve_vintage(),
```

Comment:
not 100% sure why this is needed, but the datatype before was `datetime64[ns, UTC]` which made the scraper fail on insert. This just removes the UTC

Reply:
Hrm.  That's really strange.  From a quick search, we have ~50 scrapers that have code like this and seem to be working fine.  Any idea at all why this one is different? It makes me a bit nervous.

Reply:
Yeah I'm with you--I'm really not sure. It calls the `_retrieve_vintage()` method directly, so it doesn't seem like there's any reason for this to be different than the others :/ 

I can put in some more effort to try and see what's going on